bazel/patches/0001-Set-unknown-dpid-ofctl-log-to-debug.patch

```diff
@@ -0,0 +1,26 @@
+From 215a293ff0d8deaea80739a67c63cbbe47bfdf79 Mon Sep 17 00:00:00 2001
```

Comment:
Did you consider just symlnking those patches?

Reply:
From e.g., `lte/gateway/deploy/roles/magma/files/patches/0001-Set-unknown-dpid-ofctl-log-to-debug.patch`? I would have preferred to just bazelify this folder and get the patch file from there. But they use a certain folder notation that does not agree with the bazel patch mechanics. It's on my list do dig deeper into those mechanics and see if there is an option to make the original files usable.

Reply:
Figured it out - we can get rid of the additional patch folder and use the original files. Will coordinate with Krisz for the change.